beacon_node/lighthouse_network/src/rpc/mod.rs

```diff
@@ -151,10 +157,12 @@ pub struct NetworkParams {
 /// Implements the libp2p `NetworkBehaviour` trait and therefore manages network-level
 /// logic.
 pub struct RPC<Id: ReqId, E: EthSpec> {
-    /// Rate limiter
-    limiter: Option<RateLimiter>,
+    /// Rate limiter for our responses.
+    response_limiter: Option<ResponseLimiter<E>>,
```

Comment:
```suggestion
    inbound_rate_limiter: Option<ResponseLimiter<E>>,
```
I would keep the previous naming, to also still follow the naming of the configurations. But mostly what we are rate limiting are external requests right? Not our responses

Reply:
As summarized [here](https://github.com/sigp/lighthouse/pull/5923#issuecomment-2719236781), in this PR this limiter is limiting our responses, so I think `response_limiter` is a more intuitive name than one based on `inbound` to maintain consistency with what this limiter does. 

> to also still follow the naming of the configurations

Oh, you're right, there is a naming conflict between the limiter and the configuration... 🤔 : 

https://github.com/sigp/lighthouse/blob/753e8457610009e631f413582352085e07bc7f98/beacon_node/lighthouse_network/src/rpc/mod.rs#L181-L185